p2p/libp2p/netMessenger.go

```diff
@@ -452,7 +452,9 @@ func (netMes *networkMessenger) UnregisterMessageProcessor(topic string) error {
 
 // SendToConnectedPeer sends a direct message to a connected peer
 func (netMes *networkMessenger) SendToConnectedPeer(topic string, buff []byte, peerID p2p.PeerID) error {
-	return netMes.ds.Send(topic, buff, peerID)
+	netMes.ds.SendAsync(topic, buff, peerID)
+
+	return nil
```

Comment:
Do we still need the error?

Reply:
Not quite. It just satisfy the interface.